src/components/atoms/AvatarAtom.tsx

```diff
@@ -0,0 +1,25 @@
+import Image from "next/image";
+
+type AvatarProps = {
+  src: string, 
+  alt: string, 
+  styles?: string, 
+  width: number,
+  height: number
+}
+
+const AvatarAtom = ({ src, alt, styles, width, height }: AvatarProps) => {
+  const imageStyles = `rounded-full ${styles}`
+  
+  return (
+    <Image
+      src={src}
+      alt={alt}
+      className={imageStyles}
```

Comment:
```suggestion
      className={imageStyles}
      className={classNames('rounded-full',styles)}
```

PABotの中でも使用されている`classNames`というライブラリを使えばちょっと
簡単に書けそうです！
PABot-t3では`src/components/molecules/app-bar/index.tsx`こちらのファイルの`30行目`が参考になりそうです！